src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/Summary.tsx

```diff
@@ -61,6 +61,7 @@ export function Summary({
             {t(PurchaseOverviewTexts.summary.messageInZone)}
           </ThemeText>
         );
+      case 'multiple-stop-harbor':
```

Comment:
Just make sure that the exact text must be shown here, as when the case is `none` or if there is a different message to be presented.

Reply:
https://github.com/AtB-AS/kundevendt/issues/4044
Seems like it mght be handled here

Reply:
Fixed now